src/_guides/libraries/objective-c-interop.md

```diff
@@ -293,6 +293,44 @@ then check the status, and wait for the duration of the audio file:
     }
 ```
 
+### Callbacks and multithreading limitations
+
+While `ffigen` does support converting
+Dart functions to Objective-C blocks,
+there are some limitations to keep in mind
+when using them.
```

Comment:
This intro sounds like the points below are only about blocks, but the API threading limitations (especially the main thread limitation) apply to *all* calls, regardless of whether or not they involve callbacks.